src/ngraph_utils.cc

```diff
@@ -265,6 +265,32 @@ void MemoryProfile(long& vm_usage, long& resident_set) {
     vm_usage = vsize / 1024;   // unit kb
     resident_set = rss * page_size_kb;
   }
+}
+
+void NgraphNodeDump(const shared_ptr<ng::Node>& node,
+                    const std::shared_ptr<ngraph::Function>& ng_function,
+                    int RankID) {
```

Comment:
Why is RankID a CamelCase when other variables are snake_case? Please follow the correct coding convention.